src/app/phast/phast-report/results-data/results-data.component.ts

```diff
@@ -117,6 +135,24 @@ export class ResultsDataComponent implements OnInit {
     }
   }
 
+  convertEAFEnergyUsed(results: PhastResults) {
```

Comment:
Feel like this conversion method should be in the convertPhastService, I imagine it will have to be re-used